haskell-build-2/orb.yml

```diff
@@ -119,14 +119,19 @@ commands:
         description: Number of threads for uploaded and downloading binary cache.
         type: string
         default: ${BINARY_CACHE_THREADS}
+      binary-cache-aws-log-level:
+        description: AWS log level
+        type: string
+        default: ${BINARY_CACHE_AWS_LOG_LEVEL}
 
     steps:
       - run:
           name: Restore binary cache
           command: |
             cabal-cache sync-from-archive --archive-uri "<< parameters.binary-cache-uri >>" \
-              <<# parameters.binary-cache-region >> --region "<< parameters.binary-cache-region >>" <</ parameters.binary-cache-region >> \
-              <<# parameters.binary-cache-threads >> --threads "<< parameters.binary-cache-threads >>" <</ parameters.binary-cache-threads >>
+              <<# parameters.binary-cache-region        >> --region         "<< parameters.binary-cache-region        >>" <</ parameters.binary-cache-region        >> \
+              <<# parameters.binary-cache-threads       >> --threads        "<< parameters.binary-cache-threads       >>" <</ parameters.binary-cache-threads       >> \
+              <<# parameters.binary-cache-aws-log-level >> --aws-log-level  "<< parameters.binary-cache-aws-log-level >>" <</ parameters.binary-cache-aws-log-level >>
```

Comment:
This last line doesn't work because when the parameter is absent, it defaults to `--aws-log-level  ""` rather than leave out the command line argument as I expected.

Reply:
Failing build here: https://circleci.com/gh/haskell-works/cabal-cache/530

Reply:
Should we just have one `log-level` parameter, not specialising to AWS?

I would also expect it to skip the parameter, it is weird that it doesn't...
One thing we could try it:
```
default: ${BINARY_CACHE_AWS_LOG_LEVEL:-LogInfo}
```